<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSetupDialog</class>
 <widget class="QDialog" name="ConnectionSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parametri Connessione</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nome &amp;database:</string>
       </property>
       <property name="buddy">
        <cstring>le_dbname</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_dbname"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Indirizzo server:</string>
       </property>
       <property name="buddy">
        <cstring>le_hostname</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_hostname"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Porta del server:</string>
       </property>
       <property name="buddy">
        <cstring>le_port</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="le_port">
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_clear">
       <property name="text">
        <string>&amp;Cancella</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_save">
       <property name="text">
        <string>&amp;Salva</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>le_dbname</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>le_hostname</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>le_port</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_save</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionSetupDialog</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_save</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveSettings()</slot>
 </slots>
</ui>
